geode-management/src/main/java/org/apache/geode/cache/configuration/RegionConfig.java

```diff
@@ -362,9 +361,15 @@ public String getType() {
    * {@link String }
    *
    */
-  public void setType(RegionShortcut regionShortcut) {
-    if (regionShortcut != null) {
-      this.type = regionShortcut.name();
+  public void setType(RegionType regionType) {
+    if (regionType != null) {
+      setType(regionType.name());
+    }
+  }
+
+  public void setType(String regionType) {
+    this.type = regionType;
```

Comment:
I think it would be nice to uppercase the value here so that users can use either case.